reference/api/tasks.md

```diff
@@ -262,3 +276,65 @@ Get a single task.
 ## Cancel tasks
 
 ## Delete tasks
+
+<RouteHighlighter method="DELETE" route="/tasks?{task_uid}"/>
+
+Delete a finished (`succeeded`, `failed`, or `canceled`) task based on `uid`, `status`, `type`, `indexUid`, `canceledBy`, or date. Task deletion is an atomic transaction: **either all tasks are successfully deleted, or none are**.
+
+::: warning
+To prevent users from accidentally deleting the entire history, Meilisearch throws the [`missing_task_filters`](/reference/errors/error_codes.md#missing-task-filters) error if this route is used without any filters (DELETE `/tasks`).
```

Comment:
Not sure if necessary, just a suggestion.

```suggestion
To prevent users from accidentally deleting the entire task history, Meilisearch throws the [`missing_task_filters`](/reference/errors/error_codes.md#missing-task-filters) error if this route is used without any filters (DELETE `/tasks`).
```